docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -195,9 +195,10 @@ which can also be obtained using the ``az account list-locations`` command.
 You can also try to assign a name to an Public IP in Azure before starting
 the process, or use ``nslookup`` with the name you have in mind to check
 if it's available for use.
-In the rare chance that name in the ``data.fqdn`` field is not available,
-you must create a ConfigMap with a unique name and restart the
-MongoDB instance.
+
+In the rare chance that the name specified in the ``data.fqdn`` field is not
+available, you will need to create a ConfigMap with a unique name and
+restart the MongoDB instance.
```

Comment:
I don't think they have a MongoDB instance running at this point, so there's nothing to restart.